frontend/src/scenes/surveys/SurveyCustomization.tsx

```diff
@@ -105,6 +105,13 @@ export function Customization({ appearance, surveyQuestionItem, onAppearanceChan
                     onChange={(submitButtonTextColor) => onAppearanceChange({ ...appearance, submitButtonTextColor })}
                     disabled={!surveysStylingAvailable}
                 />
+
+                <div className="mt-2">Survey form zIndex</div>
+                <LemonInput
+                    value={appearance?.zIndex}
+                    onChange={(zIndex) => onAppearanceChange({ ...appearance, zIndex })}
+                    disabled={!surveysStylingAvailable}
```

Comment:
should we include a placeholder here?  The rest of the forms have one.

Reply:
good idea, done! 
<img width="802" alt="image" src="https://github.com/user-attachments/assets/1df0b8f7-8d24-4daf-bf7b-10dd0b352a04">

Reply:
not to be nitpicky, but maybe an example value, too?  Or maybe a tooltip explaining what a zIndex is and the units in which its measured?  I know I'm being pedantic, I'm just not sure how much a PM e.g. would know what a zIndex is in this context.

Reply:
The default is 99,999. I can add that here and put in a tooltip about what this is.

Reply:
![image](https://github.com/user-attachments/assets/b7b0fb4c-f06f-4666-8adf-8ec402c30456)